projects/1.18/assets/geolosys/geolosys/lang/zh_cn.json

```
@@ -60,34 +81,40 @@
   "item.geolosys.zinc_cluster": "锌矿簇",
   "item.geolosys.osmium_cluster": "锇矿簇",
   "item.geolosys.yellorium_cluster": "黄铀矿簇",
-  "item.geolosys.ancient_debris_cluster": "远古残骸碎块",
-  "item.geolosys.nether_gold_cluster": "贫金矿簇",
+  "item.geolosys.ancient_debris_cluster": "远古残骸矿簇",
+  "item.geolosys.nether_gold_cluster": "贫瘠金矿簇",
   "item.geolosys.tin_ingot": "锡锭",
   "item.geolosys.silver_ingot": "银锭",
   "item.geolosys.lead_ingot": "铅锭",
   "item.geolosys.aluminum_ingot": "铝锭",
   "item.geolosys.nickel_ingot": "镍锭",
   "item.geolosys.platinum_ingot": "铂锭",
   "item.geolosys.zinc_ingot": "锌锭",
-  "item.geolosys.aluminum_nugget": "铝粒",
-  "item.geolosys.copper_nugget": "铜粒",
-  "item.geolosys.lead_nugget": "铅粒",
-  "item.geolosys.nickel_nugget": "镍粒",
-  "item.geolosys.platinum_nugget": "铂粒",
-  "item.geolosys.silver_nugget": "银粒",
-  "item.geolosys.tin_nugget": "锡粒",
-  "item.geolosys.zinc_nugget": "锌粒",
+  "item.geolosys.aluminum_nugget": "铝块",
+  "item.geolosys.copper_nugget": "铜块",
+  "item.geolosys.lead_nugget": "铅块",
+  "item.geolosys.nickel_nugget": "镍块",
+  "item.geolosys.platinum_nugget": "铂金块",
+  "item.geolosys.silver_nugget": "银块",
+  "item.geolosys.tin_nugget": "锡块",
+  "item.geolosys.zinc_nugget": "锌块",
   "item.geolosys.peat_coal": "泥炭",
   "item.geolosys.lignite_coal": "褐煤",
   "item.geolosys.bituminous_coal": "烟煤",
   "item.geolosys.anthracite_coal": "无烟煤",
   "item.geolosys.lignite_coal_coke": "褐焦煤",
   "item.geolosys.bituminous_coal_coke": "烟焦煤",
   "item.geolosys.field_manual": "野外探矿手册",
-  "item.geolosys.field_manual.landing": "通过Geolosys进行勘探的指南。$(br2)$(l)$(#f00)本书正在开发$()$(br2)$(o)$(#f59b42)请记住，可用的岩层可能与整合包作者所列出的不同。$()",
-  "item.geolosys.prospectors_pick": "探矿工之镐",
-  "geolosys.pro_pick.tooltip": "和指南针搭配使用，效果更佳！",
+  "item.geolosys.field_manual.landing": "一切关于在 Geolosys 中挖矿的内容，尽在这本手册中！",
+  "item.geolosys.prospectors_pick": "探矿镐",
+  "geolosys.pro_pick.tooltip": "和指南针搭配使用效果更佳",
+  "geolosys.pro_pick.tooltip.nonefound_surface": "在这片区域中没有找到矿藏。",
   "geolosys.pro_pick.tooltip.nonefound": "在这个方向上没有找到矿藏",
   "geolosys.pro_pick.tooltip.found_surface": "在这片区域中发现 %s",
-  "geolosys.pro_pick.tooltip.found": "在你的%s方向上发现了%s"
-}
\ No newline at end of file
+  "geolosys.pro_pick.tooltip.found": "你找到了 %s 在你的 %s 方向",

```

不知道第二个%s具体内容是什么.....
如果是[方向+距离]或者[方向]：
```suggestion
  "geolosys.pro_pick.tooltip.found": "在%2$s处发现%1$s",
```
如果是[距离]：
```suggestion
  "geolosys.pro_pick.tooltip.found": "在距你%2$s处发现%1$s",
```
应该是可以用`n$`标记来换顺序的。